### PR TITLE
Issue#48 fix Capitalization fix for DID spec

### PR DESCRIPTION
**Pull request by mskuybeda**
Fixed capitalization for DID parameter

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Decentralized Identifier Resolution (DID Resolution) v0.2</title>
+    <title>Decentralized Identifier Resolution (DID resolution) v0.2</title>
```
Comment (peacekeeper) [inline index.html]: I think the title and headings should still use upper case..

File: index.html [hunk 2]
```diff
@@ -233,14 +233,14 @@ <h1>Terminology</h1>
 </section>
 
 <section id="architectures">
-	<h1>DID Resolver Architectures</h1>
-	<p class="issue">TODO: Describe how <a>DID Resolvers</a> are implemented and used, describe the relevance
+	<h1>DID resolver Architectures</h1>
```
Comment (peacekeeper) [inline index.html]: See above, headings should use uppercase.

File: index.html [hunk 3]
```diff
@@ -831,19 +831,19 @@ <h2>Examples</h2>
 </section>
 
 <section id="did-resolution-result">
-	<h1>DID Resolution Result</h1>
+	<h1>DID resolution result</h1>
```
Comment (peacekeeper) [inline index.html]: See above, headings should use uppercase.

File: index.html [hunk 4]
```diff
@@ -1136,44 +1136,44 @@ <h2>Caching</h2>
...
+		<h2>DID method governance</h2>
+		<p class="issue" data-number="6">Describe which methods a <a>DID resolver</a> should support, and potential implications.</p>
 
 	</section>
 
 </section>
 
 <section class="appendix">
-	<h1>DID Resolution Resources</h1>
+	<h1>DID resolution resources</h1>
```
Comment (peacekeeper) [inline index.html]: See above, headings should use uppercase.

File: index.html [hunk 5]
```diff
@@ -901,19 +901,19 @@ <h2>Example</h2>
 	</section>
 
 	<section id="output-diddocument">
-		<h2>DID Document</h2>
-		<p>A <a>DID Document</a> associated with a <a>DID</a>. The result of <a href="#resolving"></a>.</p>
+		<h2>DID document</h2>
```
Comment (peacekeeper) [inline index.html]: See above.

**Comment by mskuybeda** [comment]
I can. Not really sure about these one:
<<And could you also fix the remaining terms "Decentralized Identifier", "Decentralized Identifier Registry", and "Service Endpoint"?>>

Do you want to make them lower case as well?

**Comment by peacekeeper** [comment]
Yes I think those should be lowercase as well, except in titles or headings. To be consistent with the DID Core spec..

**Comment by OR13** [comment]
@peacekeeper @dmitrizagidulin suggest closing stale PR with merge conflicts.

**Comment by TallTed** [comment]
@mskuybeda -- 

If you can resolve the merge conflicts, this will likely be merged to the main branch, and your contribution recorded and appreciated. 

If you cannot resolve the merge conflicts, this will likely be closed without being merged, and your contribution lost to the sands of time.

**Comment by pchampin** [comment]
This was discussed during the WG meeting on 2024-08-22:
https://www.w3.org/2024/08/22-did-minutes.html#t11

**Comment by peacekeeper** [comment]
I propose to close this, since the capitalization has been fixed some time ago, e.g. in commits https://github.com/w3c/did-resolution/commit/73d2f61bf3cd21cc6c3653486e6d3fe9fe87cced and https://github.com/w3c/did-resolution/commit/0f9c23b96c0fb2ea10196aab4e6d6501003733cf, making this PR now obsolete.

**Comment by pchampin** [comment]
This was discussed during the [#did meeting on 12 September 2024](https://www.w3.org/2024/09/12-did-minutes.html#t06).

<details><summary><i>View the transcript</i></summary>

### DID Resolution PR and Issue Processing
<cite>&lt;markus_sabadello&gt;</cite>[w3c/<wbr>did-resolution#49</wbr>](https://github.com/w3c/did-resolution/pull/49)

----
</details>

**Comment by pchampin** [comment]
This was discussed during the [#did meeting on 12 September 2024](https://www.w3.org/2024/09/12-did-minutes.html#t07).

<details><summary><i>View the transcript</i></summary>

#### [w3c/<wbr>did-resolution#49</wbr>](https://github.com/w3c/did-resolution/pull/49)<cite>markus_sabadello:</cite> this is a very old PR.
<cite>&lt;manu&gt;</cite> +1 to close
<cite>markus_sabadello:</cite> Just an editorial thing, no need to spend much time. Will marke it as "pending close"


----
</details>

**Comment by mccown** [comment]
For readability, I prefer the title case.  However, most of the standards seem to be lower case and it's better to be uniform.

**Comment by peacekeeper** [comment]
Closing, as discussed in today's DID WG meeting.

**Pull request closed without merging**